recipes/simpleaudio/recipe.yaml

```diff
@@ -0,0 +1,51 @@
+context:
+  name: simpleaudio
+  version: "1.0.4"
+  python_min: "3.6"
```

Comment:
don't bother with `python_min`: at present, `conda-forge` is only going to even _attempt_ to build for `["3.9","3.10","3.11","3.12","3.13"]`, and it _will_ change in the future (e.g. with the release of `3.14`, `3.9` will be dropped)